core/ops.rs

```diff
@@ -139,7 +140,13 @@ impl OpCtx {
       // SAFETY: all arguments are coming from the trait and they have
       // static lifetime
       let c_fn = unsafe {
-        CFunctionInfo::new(args.as_ptr(), fast_fn.args.len(), ret.as_ptr())
+        CFunctionInfo::new(
+          args.as_ptr(),
+          fast_fn.args.len(),
+          ret.as_ptr(),
+          // TODO(bartlomieju): what's the correct value to use here?
+          Int64Representation::Number,
```

Comment:
@aapoalas what a correct value should be here?

Reply:
For now nothing changes with `::Number` (though under the hood it would now be possible to return 64 bit integers from fast calls and get them as numbers on JS side: This just assumes you fully understand that this can lead to loss of data if your number is not a safe integer, and we probably do not want to make that assumption in a blanket way so it's better that we do not start allowing 64 bit integer return values in fast calls all willy-nilly.).

Eventually, I would suggest we consider changing this to `BigInt` and changing serde_v8 to return BigInts for all 64-bit numbers. When the return value numbers are small integers this will be a slight loss in performance, but if the return value numbers are known to be small integers then why are they being returned as 64-bit integers?

When the numbers are not small integers and especially when they're unsafe integers, then returning as BigInt is much faster.